glusterd2/brick/glusterfsd.go

```diff
@@ -58,9 +58,14 @@ func (b *Glusterfsd) Args() []string {
 
 	logFile := path.Join(config.GetString("logdir"), "glusterfs", "bricks", fmt.Sprintf("%s.log", brickPathWithoutSlashes))
 
-	brickPort := strconv.Itoa(pmap.AssignPort(0, b.brickinfo.Path))
-
-	volFileID := b.brickinfo.VolfileID + "." + gdctx.MyUUID.String() + "." + brickPathWithoutSlashes
+	portBrickServer := pmap.RegistrySearch(b.brickinfo.Path, pmap.GfPmapPortBrickserver)
```

Comment:
The tests are failing. https://ci.centos.org/job/gluster_glusterd2/3267/console

Please check if it's a genuine failure. See if this change breaks something in snapshots